src/main/java/com/entropy/entropay/auth/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.entropy.entropay.auth;
+
+public class Constants {
+
+    public static final Long JWT_TOKEN_VALIDITY = Long.valueOf(5 * 60 * 60);
```

Comment:
I'll prefer to have this a property

Reply:
it'd work on the bootstrap files until config server is up